### ISSUE TITLE
Nvidia Installer broken on 6.0 Odin

### ISSUE THREAD

**Issue by selwynorren** [elementary/appcenter#1548]
## What Happened
Hi Guys,

Not sure if this is truly bug or something related to my system itself, however on Elementary OS 6.0 20210611 daily image. If I install the nvidia-driver-460 using the App Center, it install fine and well, however the Nvidia X Server settings opens to a blank window with no configuration option inside at all.

After reboot, only my primary monitor switches on and the second monitor switches off and is not detected at all.

Once i remove the driver from App Center and install using CLI with this command:
sudo ubuntu-drivers install

This still installs nvidia-driver-460 driver, but now both screens are detected and the Nvidia X Server settings shows up with proper contents.

Essentially the install candidate in App Center does not install everything needed. I have tried this will all the options listed in App Center

## Expected Behavior
Nvidia Driver to install correctly, without having to additionally use a terminal commend to install correctly.

## Steps to Reproduce
Install nvidia driver from the updates page in App Center, Reboot and try configure Nvidia Settings (Should get a blank page and potentially incorrect display windows if using a multi-monitor system like I do)

### Logs
Not sure where to find them

## Platform Information
Odin 6.0 Beta 2 with latest updates

**Comment by cassidyjames** [comment]
Getting reports of this on the RC, so I figure we can mark it as confirmed. Unsure what the fix is from our side or if this belongs on a different repo.

**Comment by davidmhewitt** [comment]
I've just opened #1581 that makes AppCenter match the behavior of `sudo ubuntu-drivers install`.

However, I noticed some interesting behavior on my desktop machine with the nvidia driver that's unrelated to AppCenter.

My machine has a single dedicated GPU and no iGPU. As expected, a clean install of elementary works fine with the nouveau driver. After installing the proprietary driver with `sudo ubuntu-drivers install` or the fixed AppCenter, I still get a blank nvidia X settings window after a reboot and `glxinfo | grep renderer` still results in `OpenGL renderer string: llvmpipe (LLVM 12.0.0, 256 bits)
`

After some investigation, it turns out that this is related to the PRIME stuff. It seems to set itself up in "on-demand" mode by default, meaning that I can seemingly dynamically switch between using the proprietary nvidia driver or not (which is super weird on a desktop where you only have 1 GPU). But by default, any applications you launch don't use the nvidia driver.

So I can run 
`__NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia glxinfo | grep renderer`

And get told I'm using the nvidia proprietary driver

And if I remove the environment variables, it uses the llvmpipe driver.

I opted to do `sudo prime-select nvidia` and a reboot to force it to use the nvidia driver all the time, which resolves the issue with the nvidia settings application.

As always, a fantastic experience with the proprietary nvidia drivers :roll_eyes:
